lib/aasm/base.rb

```diff
@@ -40,6 +42,9 @@ def initialize(klass, name, state_machine, options={}, &block)
       # Set to true to namespace reader methods and constants
       configure :namespace, false
 
+      # Configure a logger, with default being a Logger to STDERR
+      configure :logger, Logger.new(STDERR)
```

Comment:
Quick question here, do we feel that we might want to introduce a `AASM::Configuration` that can allow for global changes across all `aasm` blocks? Not sure if this is a step too far for this PR or where the direction AASM wants to go. Not even sure how the precedence of logger configurations are honored, (local => global => Kernel)